salt/environment_settings.yml

```diff
@@ -539,6 +539,10 @@ environments:
         business_unit: operations
         security_groups:
           - webapp-odl-vpn
+      elasticsearch:
+        num_instances: 5
+        size: r4.xlarge
+        data_volume_size: 800
```

Comment:
We should probably set this to 1 or 1.5 TB since we will be starting to throw metric data at this cluster.

Reply:
✅